R/mrgvalidate.R

```diff
@@ -7,7 +7,7 @@
 #' `.docx` format. Specifications for input data can be found on the
 #' [input_formats] help page. The companion `mrgvalprep` package contains
 #' helpers for formatting input data to the specification in [input_formats].
-#' @importFrom rlang abort warn inform %||%
+#' @importFrom rlang abort warn inform %||% .data
 #' @importFrom lifecycle badge
```

Comment:
I saw that you removed `.data` as an import in the other locations. What was the reasoning behind adding the import here if we're removing its use from everywhere else?

Reply:
Edit: I searched for `.data` in the repo and it just took forever to show me the other locations. I see that its still used elsewhere now